docs/guides/signature-request.md

```diff
@@ -1,21 +1,19 @@
 # Signature request
 
-Client can invite other people to sign documents by providing them with secure document URLs
-containing a secret key. The secret key will decrypt the document before signing, ensuring that
-Certifaction never access the document in clear text.
+Users can invite other people to sign documents by providing them with secure signature request URLs, which contain a secret key. The secret key will decrypt the document before signing, ensuring that no third-party (not even Certifaction) has access to the document in clear text.
 
-Certifaction document signature request rely on the Certifaction Digital Archive to store end-to-end encrypted
-document.
+Certifaction document signature requests rely on the Certifaction Digital Archive to store end-to-end encrypted
+documents.
 
 During a signature request, the CLI will:
 
-1.  Receive the PDF document to sign and the signer information
-2.  Process the document, adding security features including the secure Digital Twin URL
-3.  Encrypt the document using either the secret key contained in the Digital Twin URL, or a manually provided key
-4.  Store the document in Certifaction Digital Archive for later retrieval
-5.  Return the secure URL.
+1.  Receive the PDF document to sign and the signer information.
+2.  Process the document, adding security features including the secure Digital Archive URL.
```

Comment:
Maybe "the secure Digital Archive URL" --> "a Digital Archive URL" because it is the first time that we mention it?  It means that we would need a glossary at one point.